config/main.py

```diff
@@ -2092,7 +2092,7 @@ def delete(counter_name, verbose):
 #
 # 'add_reasons' subcommand ('config dropcounters add_reasons')
 #
-@dropcounters.command()
+@dropcounters.command('add_reasons')
```

Comment:
is this release as well? I suggest to change to add-reasons

Reply:
Changed.